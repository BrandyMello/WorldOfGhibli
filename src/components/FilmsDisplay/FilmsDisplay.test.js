import React from 'react';
import FilmsDisplay from './FilmsDisplay';
import { shallow } from 'enzyme';

describe('FilmsDisplay', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(
      <FilmsDisplay 
      films={[{id: 1, name: 'Movie1'}, {id: 2, name: 'Movie2'}]}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});