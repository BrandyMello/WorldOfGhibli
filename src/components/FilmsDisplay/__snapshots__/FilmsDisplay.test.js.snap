// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsDisplay should match the snapshot 1`] = `
<div
  className="movie-container"
>
  <Film
    key="1"
    photo="https://fesapusewebsite.blob.core.windows.net/fathom/castleinthesky-1000x1480-r3-f6aba65dfc41c4fc47fc9289c1674057.jpg"
  />
  <Film
    key="2"
    photo="http://barkerhost.com/wp-content/uploads/sites/4/2015/04/bwVhmPpydv8P7mWfrmL3XVw0MV5-0.jpg"
  />
</div>
`;
